predicators/datasets/vlm_input_data_prompts/vision_api/types_burger.txt

```diff
@@ -0,0 +1,16 @@
+_object_type = Type("object", [])
```

Comment:
I think this way of doing things is fine, but I wonder if we could dynamically generate this text from our types in every env automatically? Did you try this/do you think it's possible? That way, we wouldn't have to create a new file like this for every env/change it whenever the env changes.